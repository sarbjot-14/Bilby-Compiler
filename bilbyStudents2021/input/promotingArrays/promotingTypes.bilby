main{

	
	imm i := [#g,12,13.5,14];
	print i $n;
	
	imm a := [#g,#I,#F];
	print a $n;
	
	
	imm d := [6,5.6];
	print d $n;
	
	
}