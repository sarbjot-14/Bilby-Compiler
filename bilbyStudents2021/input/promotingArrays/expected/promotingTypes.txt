[103.000000, 12.000000, 13.500000, 14.000000]
[g, I, F]
[6.000000, 5.600000]
[97, 3, 4]
